plugins/org.wso2.developerstudio.eclipse.esb.project/src/org/wso2/developerstudio/eclipse/esb/project/store/handlers/ConnectorStoreEarlyStartupHandler.java

```diff
@@ -0,0 +1,109 @@
+*  Version 2.0 (the "License"); you may not use this file except
+*  in compliance with the License.
+*  You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing,
+* software distributed under the License is distributed on an
+* "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+* KIND, either express or implied.  See the License for the
+* specific language governing permissions and limitations
+* under the License.
+*/
+package org.wso2.developerstudio.eclipse.esb.project.store.handlers;
+
+import java.io.File;
+import java.io.IOException;
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.net.URL;
+
+import org.eclipse.core.runtime.FileLocator;
+import org.eclipse.core.runtime.Platform;
+import org.eclipse.jetty.servlet.DefaultServlet;
+import org.eclipse.jetty.servlet.ServletContextHandler;
+import org.eclipse.ui.IStartup;
+import org.osgi.framework.Bundle;
+import org.wso2.developerstudio.eclipse.esb.project.Activator;
+import org.wso2.developerstudio.eclipse.esb.project.servlets.ConnectorDownloadServlet;
+import org.wso2.developerstudio.eclipse.esb.project.servlets.ConnectorRetrievalServlet;
+import org.wso2.developerstudio.eclipse.esb.project.servlets.ConnectorSearchServlet;
+import org.wso2.developerstudio.eclipse.logging.core.IDeveloperStudioLog;
+import org.wso2.developerstudio.eclipse.logging.core.Logger;
+import org.wso2.developerstudio.eclipse.templates.dashboard.handlers.JettyServerHandler;
+
+/**
+ * Binds new servlets to paths of the jetty server
+ * 
+ */
+public class ConnectorStoreEarlyStartupHandler implements IStartup {
+
+    private static IDeveloperStudioLog log = Logger.getLog(Activator.PLUGIN_ID);
+    private static final String CONTEXT_PATH = "/project/connectors";
+    private static final String CONNECTOR_RETRIEVAL_SERVLET_PATH = "/list";
+    private static final String CONNECTOR_DOWNLOAD_SERVLET_PATH = "/download";
+    private static final String CONNECTOR_SEARCH_SERVLET_PATH = "/search";
+    private static final String WEB_APP_LOCATION = "HTMLPages";
+
+    /*
+     * (non-Javadoc)
+     * 
+     * @see org.eclipse.ui.IStartup#earlyStartup()
+     */
+    @Override
+    public void earlyStartup() {
+        JettyServerHandler jettyServerHandler = JettyServerHandler.getInstance();
+
+        // Registering servlet context handler of store connectors window
+        ServletContextHandler connectorsContext = new ServletContextHandler();
+
+        // Context path where static webpages are hosted
+        String webAppPath = "";
+        try {
+            // Get web app path from current bundle
+            webAppPath = getWebAppPath();
+        } catch (URISyntaxException uriException) {
+            log.error("Error resolving web app path", uriException);
+        } catch (IOException ioException) {
+            log.error("Error resolving web app path", ioException);
```

Comment:
We can catch both IOException and URISyntaxException in a single catch block?

Reply:
Fixed in https://github.com/wso2/devstudio-tooling-esb/pull/1063